docs/Course Wiki/ECE Course Offerings/ECE329.md

```diff
@@ -30,9 +30,9 @@ The course overall starts off quite light, especially if the student is already
 
 During the semester, the course assigns 14 written homeworks, due every week. These homework sets are generally not too difficult and will contain both calculations and derivations of expressions. Students have cited that each week's homework can take very different amounts of time, from under an hour to upwards of five hours to complete. They can sometimes be surprisingly annoying.
 
-The course will also have three midterms and a final. Difficulty of the midterms depends heavily on the professor who is organizing the course during that semester. There are only a few practice exams - thus it is recommended to keep up with the homework every week. Professor Kudeki's notes have good practice questions with solutions in them, should the student find that they need more practice. Students are either provided a cheatsheet or allowed to bring index card(s) to the exam.
+The course will also have three midterms and a final. Difficulty of the midterms depends heavily on the professor who is organizing the course during that semester. There are only a few practice exams - thus it is recommended to keep up with the homework every week. Professor Kudeki's notes have good practice questions with solutions in them, should the student find that they need more practice. Cheatsheet policies vary depending on the semester and the lead instructor. During some semesters, students are provided a cheatsheet. During others, students are allowed to bring an index card (3x5 inches) front and back to the exam, as well as index cards from prior midterms. For example, for midterm 3 you would be allowed three index cards front and back. You are highly recommended to reuse index cards from prior midterms, but this is not enforced.
```

Comment:
still unclear. do those three index cards have to be the exact ones you brought to prior midterms, or can they be any notecards. This should be worded either as:
1. "During others, students are allowed to bring one double-sided 3x5 index card to the first exam, two double-sided 3x5 index cards to the second exam, and so on"
OR
2. "During others, students are allowed to bring one double-sided 3x5 index card to each exam, along with the notecards they brought to any previous exams (note: these must be the *same* notecards, and does not mean that for the second midterm you can bring two notecards with only midterm 2 content)"
3.